### DISCUSSION TITLE
How to get the upload date of videos on a channel?

### DISCUSSION THREAD

**Discussion by HubKing** [Tyrrrz/YoutubeExplode#586] [Q&A]
Getting the videos of a channel used to return a collection of `Video` that has `UploadDate`. After updating to the latest version of this library, now it returns a collection of `PlayListVideo` which does not have `UploadDate`. What gives? Why doesn't it have the upload date? How can I then know the upload date of video on a channel?

**Comment by Tyrrrz** [comment]
YouTube does not return upload dates anymore for videos inside playlists

**Reply by HubKing** [reply]
Thanks. I want to mark it as the answer, but somehow the button does not work. In the browser's console, I see 422 error response. Probably GitHub bug.
